src/System.Net.Security/src/Resources/Strings.resx

```diff
@@ -441,4 +441,7 @@
   <data name="net_nego_server_not_supported" xml:space="preserve">
     <value>Server implementation is not supported</value>
   </data>
+  <data name="net_nego_protection_level_not_supported" xml:space="preserve">
+    <value>Requested protection level is not supported</value>
```

Comment:
Nit: these messages are full sentences and should end with periods.  Also, can we say something more about why it's not supported, whether there's something the user can do to make it supported, etc.?